todo/src/App.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './App.css';
+import TodoList from './Components/TodoList';
+
+class App extends Component {
```

Comment:
love seeing an app.js bare and simple. good job keeping everything necessary in the components.